src/protocol/Runtime.coffee

```diff
@@ -47,16 +48,19 @@ class RuntimeProtocol
 
       if network.isStarted()
         # processes don't exist until started
+        @emit 'ports', portsPayload name, network.graph
```

Comment:
Please use parentheses to disambiguate these function calls (also for the other instances in the file). It is not obvious which arguments go to which function...